.github/workflows/pythonpackage.yml

```diff
@@ -34,9 +41,11 @@ jobs:
    #     flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
    #     flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+  
+    - name: Run migrations
+      run: python manage.py migrate
     - name: Test with pytest
       run: |
         pip install pytest
         pip install pytest-django
-        pytest -h
         pytest
```

Comment:
This reminds me... We are using pytest in here in CI, but it's not in requirements.txt . I think it should be in requirements.txt.

Reply:
added to requirements. We should probably think about splitting requirements into dev/test/prod etc?